Errors When Linking is Running on iOS Build

Followed all the instructions for installation and configuration but am getting the following build errors when trying to build the project after installing RNShareMenu.  Any ideas on what could be going wrong?

```
Undefined symbols for architecture arm64:
  "static Foundation.URL._unconditionallyBridgeFromObjectiveC(__C.NSURL?) -> Foundation.URL", referenced from:
      @objc static RNShareMenu.ShareMenuMessenger.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenuMessenger.o)
  "Foundation.URL.scheme.getter : Swift.String?", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "type metadata accessor for Foundation.URL", referenced from:
      @objc static RNShareMenu.ShareMenuMessenger.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenuMessenger.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func10 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.initialShare.unsafeMutableAddressor : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined init with copy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.setter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined assign with take of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any]) in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "ObjectiveC._convertBoolToObjCBool(Swift.Bool) -> ObjectiveC.ObjCBool", referenced from:
      @objc static RNShareMenu.ShareMenu.requiresMainQueueSetup() -> Swift.Bool in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "associated type descriptor for _ObjectiveCType", referenced from:
      l_got.$s15_ObjectiveCTypes01_A11CBridgeablePTl in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$s15_ObjectiveCTypes01_A11CBridgeablePTl in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "associated type descriptor for RawValue", referenced from:
      l_got.$s8RawValueSYTl in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$s8RawValueSYTl in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "value witness table for Builtin.UnknownObject", referenced from:
      full type metadata for RNShareMenu.ShareMenuMessenger in libRNShareMenu.a(ShareMenuMessenger.o)
      full type metadata for __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenuMessenger.o)
      full type metadata for __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      full type metadata for RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Foundation):Swift.Dictionary._bridgeToObjectiveC() -> __C.NSDictionary", referenced from:
      RNShareMenu.ShareMenu.dispatchEvent(with: [Swift.String : Swift.String]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "static (extension in Foundation):Swift.Dictionary._unconditionallyBridgeFromObjectiveC(__C.NSDictionary?) -> [A : B]", referenced from:
      @objc static RNShareMenu.ShareMenuMessenger.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenuMessenger.o)
  "type metadata accessor for Swift.Dictionary", referenced from:
      type metadata accessor for [__C.UIApplicationOpenURLOptionsKey : Any] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for [Swift.String : Swift.String] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int", referenced from:
      l_got.$sSH13_rawHashValue4seedS2i_tFTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSH13_rawHashValue4seedS2i_tFTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift.Hashable.hash(into: inout Swift.Hasher) -> ()", referenced from:
      l_got.$sSH4hash4intoys6HasherVz_tFTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSH4hash4intoys6HasherVz_tFTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift.Hashable.hashValue.getter : Swift.Int", referenced from:
      l_got.$sSH9hashValueSivgTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSH9hashValueSivgTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol descriptor for Swift.Hashable", referenced from:
      l_got.$sSHMp in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSHMp in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "base conformance descriptor for Swift.Hashable: Swift.Equatable", referenced from:
      l_got.$sSHSQTb in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSHSQTb in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for static Swift.Equatable.== infix(A, A) -> Swift.Bool", referenced from:
      l_got.$sSQ2eeoiySbx_xtFZTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSQ2eeoiySbx_xtFZTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol descriptor for Swift.Equatable", referenced from:
      l_got.$sSQMp in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSQMp in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Foundation):Swift.String._bridgeToObjectiveC() -> __C.NSString", referenced from:
      __C.UIApplicationOpenURLOptionsKey.init(rawValue: Swift.String) -> __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenuMessenger.o)
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.dispatchEvent(with: [Swift.String : Swift.String]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      __C.UIApplicationOpenURLOptionsKey.init(rawValue: Swift.String) -> __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      @nonobjc __C.NSUserDefaults.init(suiteName: Swift.String?) -> __C.NSUserDefaults? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "static (extension in Foundation):Swift.String._unconditionallyBridgeFromObjectiveC(__C.NSString?) -> Swift.String", referenced from:
      __C.UIApplicationOpenURLOptionsKey.rawValue.getter : Swift.String in libRNShareMenu.a(ShareMenuMessenger.o)
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      __C.UIApplicationOpenURLOptionsKey.rawValue.getter : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.String.init(stringInterpolation: Swift.DefaultStringInterpolation) -> Swift.String", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.String.init(_builtinStringLiteral: Builtin.RawPointer, utf8CodeUnitCount: Builtin.Word, isASCII: Builtin.Int1) -> Swift.String", referenced from:
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func0 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func1 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func2 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func3 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func4 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func5 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func6 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "static Swift.String.== infix(Swift.String, Swift.String) -> Swift.Bool", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "type metadata for Swift.String", referenced from:
      lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift._ObjectiveCBridgeable in Foundation in libRNShareMenu.a(ShareMenuMessenger.o)
      RNShareMenu.ShareMenu.supportedEvents() -> [Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      @objc RNShareMenu.ShareMenu.supportedEvents() -> [Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for [Swift.String : Swift.String] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for [Swift.String] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.dispatchEvent(with: [Swift.String : Swift.String]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "protocol witness table for Swift.String : Swift.Hashable in Swift", referenced from:
      protocol witness for Swift.Hashable.hashValue.getter : Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift.Hashable.hash(into: inout Swift.Hasher) -> () in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable? in conformance __C.UIApplicationOpenURLOptionsKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      type metadata accessor for [Swift.String : Swift.String] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.dispatchEvent(with: [Swift.String : Swift.String]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      protocol witness for Swift.Hashable.hashValue.getter : Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "protocol witness table for Swift.String : Swift.Equatable in Swift", referenced from:
      protocol witness for static Swift.Equatable.== infix(A, A) -> Swift.Bool in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Equatable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for static Swift.Equatable.== infix(A, A) -> Swift.Bool in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Equatable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol witness table for Swift.String : Swift.TextOutputStreamable in Swift", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol conformance descriptor for Swift.String : Swift._ObjectiveCBridgeable in Foundation", referenced from:
      lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift._ObjectiveCBridgeable in Foundation in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift._ObjectiveCBridgeable in Foundation in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol witness table for Swift.String : Swift.CustomStringConvertible in Swift", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift.RawRepresentable.rawValue.getter : A.RawValue", referenced from:
      l_got.$sSY8rawValue03RawB0QzvgTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSY8rawValue03RawB0QzvgTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift.RawRepresentable.init(rawValue: A.RawValue) -> A?", referenced from:
      l_got.$sSY8rawValuexSg03RawB0Qz_tcfCTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSY8rawValuexSg03RawB0Qz_tcfCTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol descriptor for Swift.RawRepresentable", referenced from:
      l_got.$sSYMp in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$sSYMp in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Foundation):Swift.Array._bridgeToObjectiveC() -> __C.NSArray", referenced from:
      @objc RNShareMenu.ShareMenu.supportedEvents() -> [Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      reabstraction thunk helper from @callee_unowned @convention(block) (@unowned __C.NSArray?) -> () to @escaping @callee_guaranteed (@guaranteed [Any]?) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.Array.init(arrayLiteral: A...) -> [A]", referenced from:
      RNShareMenu.ShareMenu.supportedEvents() -> [Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.getSharedText(callback: ([Any]?) -> ()) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "type metadata accessor for Swift.Array", referenced from:
      type metadata accessor for [__C.NSDictionary] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for [Swift.String] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol conformance descriptor for [A] : Swift.Collection in Swift", referenced from:
      lazy protocol witness table accessor for type [__C.NSDictionary] and conformance [A] : Swift.Collection in Swift in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      lazy protocol witness table accessor for type [Swift.String] and conformance [A] : Swift.Collection in Swift in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Swift):Swift.Collection.first.getter : A.Element?", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "type metadata accessor for Swift.Optional", referenced from:
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func10 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.initialShare.unsafeMutableAddressor : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined init with copy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.setter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined assign with take of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.init() -> RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.messageShare(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "Swift._bridgeAnyObjectToAny(Swift.AnyObject?) -> Any", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift._assertionFailure(_: Swift.StaticString, _: Swift.String, file: Swift.StaticString, line: Swift.UInt, flags: Swift.UInt32) -> Swift.Never", referenced from:
      static RNShareMenu.ShareMenu.shared.getter : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.targetUrlScheme.getter : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol descriptor for Swift._SwiftNewtypeWrapper", referenced from:
      l_got.$ss20_SwiftNewtypeWrapperMp in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss20_SwiftNewtypeWrapperMp in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "base conformance descriptor for Swift._SwiftNewtypeWrapper: Swift.RawRepresentable", referenced from:
      l_got.$ss20_SwiftNewtypeWrapperPSYTb in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss20_SwiftNewtypeWrapperPSYTb in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "base conformance descriptor for Swift._SwiftNewtypeWrapper: Swift._HasCustomAnyHashableRepresentation", referenced from:
      l_got.$ss20_SwiftNewtypeWrapperPs35_HasCustomAnyHashableRepresentationTb in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss20_SwiftNewtypeWrapperPs35_HasCustomAnyHashableRepresentationTb in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.RawValue: Swift.Hashable>.hashValue.getter : Swift.Int", referenced from:
      protocol witness for Swift.Hashable.hashValue.getter : Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift.Hashable.hashValue.getter : Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.RawValue: Swift.Hashable>._rawHashValue(seed: Swift.Int) -> Swift.Int", referenced from:
      protocol witness for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.RawValue: Swift.Hashable>._toCustomAnyHashable() -> Swift.AnyHashable?", referenced from:
      protocol witness for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable? in conformance __C.UIApplicationOpenURLOptionsKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable? in conformance __C.UIApplicationOpenURLOptionsKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.RawValue: Swift.Hashable>.hash(into: inout Swift.Hasher) -> ()", referenced from:
      protocol witness for Swift.Hashable.hash(into: inout Swift.Hasher) -> () in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift.Hashable.hash(into: inout Swift.Hasher) -> () in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "static (extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._forceBridgeFromObjectiveC(_: A.RawValue._ObjectiveCType, result: inout A?) -> ()", referenced from:
      protocol witness for static Swift._ObjectiveCBridgeable._forceBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> () in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for static Swift._ObjectiveCBridgeable._forceBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> () in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "static (extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._conditionallyBridgeFromObjectiveC(_: A.RawValue._ObjectiveCType, result: inout A?) -> Swift.Bool", referenced from:
      protocol witness for static Swift._ObjectiveCBridgeable._conditionallyBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> Swift.Bool in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for static Swift._ObjectiveCBridgeable._conditionallyBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> Swift.Bool in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "static (extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._unconditionallyBridgeFromObjectiveC(A.RawValue._ObjectiveCType?) -> A", referenced from:
      protocol witness for static Swift._ObjectiveCBridgeable._unconditionallyBridgeFromObjectiveC(A._ObjectiveCType?) -> A in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for static Swift._ObjectiveCBridgeable._unconditionallyBridgeFromObjectiveC(A._ObjectiveCType?) -> A in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "(extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._bridgeToObjectiveC() -> A.RawValue._ObjectiveCType", referenced from:
      protocol witness for Swift._ObjectiveCBridgeable._bridgeToObjectiveC() -> A._ObjectiveCType in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for Swift._ObjectiveCBridgeable._bridgeToObjectiveC() -> A._ObjectiveCType in conformance __C.UIApplicationOpenURLOptionsKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol descriptor for Swift._ObjectiveCBridgeable", referenced from:
      l_got.$ss21_ObjectiveCBridgeableMp in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss21_ObjectiveCBridgeableMp in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for static Swift._ObjectiveCBridgeable._forceBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> ()", referenced from:
      l_got.$ss21_ObjectiveCBridgeableP016_forceBridgeFromA1C_6resulty01_A5CTypeQz_xSgztFZTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss21_ObjectiveCBridgeableP016_forceBridgeFromA1C_6resulty01_A5CTypeQz_xSgztFZTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for static Swift._ObjectiveCBridgeable._conditionallyBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> Swift.Bool", referenced from:
      l_got.$ss21_ObjectiveCBridgeableP024_conditionallyBridgeFromA1C_6resultSb01_A5CTypeQz_xSgztFZTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss21_ObjectiveCBridgeableP024_conditionallyBridgeFromA1C_6resultSb01_A5CTypeQz_xSgztFZTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for static Swift._ObjectiveCBridgeable._unconditionallyBridgeFromObjectiveC(A._ObjectiveCType?) -> A", referenced from:
      l_got.$ss21_ObjectiveCBridgeableP026_unconditionallyBridgeFromA1Cyx01_A5CTypeQzSgFZTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss21_ObjectiveCBridgeableP026_unconditionallyBridgeFromA1Cyx01_A5CTypeQzSgFZTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift._ObjectiveCBridgeable._bridgeToObjectiveC() -> A._ObjectiveCType", referenced from:
      l_got.$ss21_ObjectiveCBridgeableP09_bridgeToA1C01_A5CTypeQzyFTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss21_ObjectiveCBridgeableP09_bridgeToA1C01_A5CTypeQzyFTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.DefaultStringInterpolation.appendInterpolation<A where A: Swift.CustomStringConvertible, A: Swift.TextOutputStreamable>(A) -> ()", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.DefaultStringInterpolation.appendLiteral(Swift.String) -> ()", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.DefaultStringInterpolation.init(literalCapacity: Swift.Int, interpolationCount: Swift.Int) -> Swift.DefaultStringInterpolation", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift._allocateUninitializedArray<A>(Builtin.Word) -> ([A], Builtin.RawPointer)", referenced from:
      RNShareMenu.ShareMenu.supportedEvents() -> [Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.getSharedText(callback: ([Any]?) -> ()) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.== infix<A where A: Swift.RawRepresentable, A.RawValue: Swift.Equatable>(A, A) -> Swift.Bool", referenced from:
      protocol witness for static Swift.Equatable.== infix(A, A) -> Swift.Bool in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Equatable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      protocol witness for static Swift.Equatable.== infix(A, A) -> Swift.Bool in conformance __C.UIApplicationOpenURLOptionsKey : Swift.Equatable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "protocol descriptor for Swift._HasCustomAnyHashableRepresentation", referenced from:
      l_got.$ss35_HasCustomAnyHashableRepresentationMp in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss35_HasCustomAnyHashableRepresentationMp in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "method descriptor for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable?", referenced from:
      l_got.$ss35_HasCustomAnyHashableRepresentationP03_tobcD0s0cD0VSgyFTq in libRNShareMenu.a(ShareMenuMessenger.o)
      l_got.$ss35_HasCustomAnyHashableRepresentationP03_tobcD0s0cD0VSgyFTq in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "Swift.print(_: Any..., separator: Swift.String, terminator: Swift.String) -> ()", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
     (maybe you meant: default argument 2 of Swift.print(_: Any..., separator: Swift.String, terminator: Swift.String) -> (), default argument 1 of Swift.print(_: Any..., separator: Swift.String, terminator: Swift.String) -> () )
  "type metadata for Any", referenced from:
      @objc static RNShareMenu.ShareMenuMessenger.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenuMessenger.o)
      type metadata accessor for [__C.UIApplicationOpenURLOptionsKey : Any] in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.getSharedText(callback: ([Any]?) -> ()) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      reabstraction thunk helper from @callee_unowned @convention(block) (@unowned __C.NSArray?) -> () to @escaping @callee_guaranteed (@guaranteed [Any]?) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "__swift_FORCE_LOAD_$_swiftCompatibility50", referenced from:
      __swift_FORCE_LOAD_$_swiftCompatibility50_$_RNShareMenu in libRNShareMenu.a(ShareMenuMessenger.o)
      __swift_FORCE_LOAD_$_swiftCompatibility50_$_RNShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
     (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibility50_$_RNShareMenu)
  "__swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements", referenced from:
      __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_RNShareMenu in libRNShareMenu.a(ShareMenuMessenger.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_RNShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
     (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_RNShareMenu)
  "_swift_beginAccess", referenced from:
      static RNShareMenu.ShareMenu._shared.getter : RNShareMenu.ShareMenu? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.shared.getter : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.getter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.setter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.modify : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.getter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.setter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_bridgeObjectRelease", referenced from:
      @objc static RNShareMenu.ShareMenuMessenger.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenuMessenger.o)
      __C.UIApplicationOpenURLOptionsKey.init(rawValue: Swift.String) -> __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenuMessenger.o)
      RNShareMenu.ShareMenu.sharedData.setter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined assign with take of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any]) in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined consume of Swift.String? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_bridgeObjectRetain", referenced from:
      __C.UIApplicationOpenURLOptionsKey.init(rawValue: Swift.String) -> __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenuMessenger.o)
      key path setter for RNShareMenu.ShareMenu.sharedData : [Swift.String : Swift.String]? : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.getter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.setter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined init with copy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined copy of Swift.String? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.supportedEvents() -> [Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_checkMetadataState", referenced from:
      outlined init with copy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined assign with take of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any]) in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.init() -> RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.messageShare(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_dynamicCast", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_endAccess", referenced from:
      static RNShareMenu.ShareMenu._shared.getter : RNShareMenu.ShareMenu? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.shared.getter : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.getter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.setter : [Swift.String : Swift.String]? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.sharedData.modify : [Swift.String : Swift.String]? with unmangled suffix ".resume.0" in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.getter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.setter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_getForeignTypeMetadata", referenced from:
      type metadata accessor for __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenuMessenger.o)
      type metadata accessor for __C.UIApplicationOpenURLOptionsKey in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_getInitializedObjCClass", referenced from:
      type metadata accessor for RNShareMenu.ShareMenuMessenger in libRNShareMenu.a(ShareMenuMessenger.o)
      type metadata accessor for RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for __C.UIApplication in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for __C.NSUserDefaults in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for __C.NSDictionary in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_getObjCClassFromMetadata", referenced from:
      __C.NSUserDefaults.__allocating_init(suiteName: Swift.String?) -> __C.NSUserDefaults? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_getObjCClassMetadata", referenced from:
      @objc static RNShareMenu.ShareMenuMessenger.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenuMessenger.o)
      @objc static RNShareMenu.ShareMenu.shared.getter : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for __C.UIApplication in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      @objc static RNShareMenu.ShareMenu.requiresMainQueueSetup() -> Swift.Bool in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for __C.NSUserDefaults in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      type metadata accessor for __C.NSDictionary in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_getTupleTypeMetadata3", referenced from:
      _globalinit_33_AE30FAC0B9FB26090D38533F91697B34_func10 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.initialShare.unsafeMutableAddressor : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined init with copy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      static RNShareMenu.ShareMenu.initialShare.setter : (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined assign with take of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any]) in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      outlined destroy of (__C.UIApplication, Foundation.URL, [__C.UIApplicationOpenURLOptionsKey : Any])? in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_getWitnessTable", referenced from:
      lazy protocol witness table accessor for type __C.UIApplicationOpenURLOptionsKey and conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type __C.UIApplicationOpenURLOptionsKey and conformance __C.UIApplicationOpenURLOptionsKey : Swift.RawRepresentable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type __C.UIApplicationOpenURLOptionsKey and conformance __C.UIApplicationOpenURLOptionsKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type __C.UIApplicationOpenURLOptionsKey and conformance __C.UIApplicationOpenURLOptionsKey : Swift.Equatable in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type __C.UIApplicationOpenURLOptionsKey and conformance __C.UIApplicationOpenURLOptionsKey : Swift._SwiftNewtypeWrapper in __C_Synthesized in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift._ObjectiveCBridgeable in Foundation in libRNShareMenu.a(ShareMenuMessenger.o)
      lazy protocol witness table accessor for type __C.UIApplicationOpenURLOptionsKey and conformance __C.UIApplicationOpenURLOptionsKey : Swift.Hashable in __C_Synthesized in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_isaMask", referenced from:
      key path getter for RNShareMenu.ShareMenu.sharedData : [Swift.String : Swift.String]? : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      key path setter for RNShareMenu.ShareMenu.sharedData : [Swift.String : Swift.String]? : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      key path getter for RNShareMenu.ShareMenu.hasListeners : Swift.Bool : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      key path setter for RNShareMenu.ShareMenu.hasListeners : Swift.Bool : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      key path getter for RNShareMenu.ShareMenu._targetUrlScheme : Swift.String? : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      key path setter for RNShareMenu.ShareMenu._targetUrlScheme : Swift.String? : RNShareMenu.ShareMenu in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.targetUrlScheme.getter : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_once", referenced from:
      RNShareMenu.NO_URL_TYPES_ERROR_MESSAGE.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.NO_URL_SCHEMES_ERROR_MESSAGE.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.NO_SCHEME_ERROR_MESSAGE.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.NO_APP_GROUP_ERROR.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.USER_DEFAULTS_KEY.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.URL_SCHEME_INFO_PLIST_KEY.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.MIME_TYPE_KEY.unsafeMutableAddressor : Swift.String in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      ...
  "_swift_release", referenced from:
      ___swift_destroy_boxed_opaque_existential_0 in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_slowAlloc", referenced from:
      ___swift_allocate_value_buffer in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
  "_swift_unknownObjectRelease", referenced from:
      RNShareMenu.ShareMenu.share(application: __C.UIApplication, openUrl: Foundation.URL, options: [__C.UIApplicationOpenURLOptionsKey : Any]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
      RNShareMenu.ShareMenu.dispatchEvent(with: [Swift.String : Swift.String]) -> () in libRNShareMenu.a(ShareMenu-E3B576B55F6870B8.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Showing first 200 notices only



Build failed    7/29/20, 12:19 PM    109.1 seconds
```

--- a/README.md
+++ b/README.md
@@ -108,6 +108,55 @@ On the pop-up, select `node_modules/react-native-share-menu/ios/ShareViewControl
 
 ![Add View Controller](screenshots/Xcode-04.png)
 
+Make these changes to your Podfile:
+
+```diff
+target '<PROJECT_NAME>' do
+  config = use_native_modules!
+
+  use_react_native!(:path => config["reactNativePath"])
+
+  target '<PROJECT_NAME>Tests' do
+    inherit! :complete
+    # Pods for testing
+  end
+
+  # Enables Flipper.
+  #
+  # Note that if you have use_frameworks! enabled, Flipper will not work and
+  # you should disable these next few lines.
+  use_flipper!
+  post_install do |installer|
+    flipper_post_install(installer)
++    installer.pods_project.targets.each do |target|
++      target.build_configurations.each do |config|
++        config.build_settings['APPLICATION_EXTENSION_API_ONLY'] = 'NO'
++      end
++    end
+  end
+end
+
++target '<SHARE_EXTENSION_NAME>' do
++  config = use_native_modules!
++
++  use_react_native!(:path => config["reactNativePath"])
++end
+```
+
+Run `pod install` in your `ios/` directory.
+
+<details>
+<summary>If you're using React Native < 0.62</summary>
+<br>
+Create a bridging header by right clicking on your project folder:
+
+- Selecting Add Files to "PROJECT_NAME"
+- Choose Swift
+- Make sure the selected target is your main app target
+- Create file and say yes to creating a Bridging Header file
+- Delete everything in the Swift file, but keep the file around
+</details>
+
 Create an App Group to be able to share data between your extension and your app. To do so, go to your app target's settings, go to `Signing & Capabilities`, press `+ Capability` and select `App Groups`
 
 ![Add App Groups](screenshots/Xcode-05.png)
@@ -185,43 +234,6 @@ Finally, in your `AppDelegate.m` add the following:
 
 If you want a custom sharing view, do these steps:
 
-Make these changes to your Podfile:
-
-```diff
-target '<PROJECT_NAME>' do
-  config = use_native_modules!
-
-  use_react_native!(:path => config["reactNativePath"])
-
-  target '<PROJECT_NAME>Tests' do
-    inherit! :complete
-    # Pods for testing
-  end
-
-  # Enables Flipper.
-  #
-  # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable these next few lines.
-  use_flipper!
-  post_install do |installer|
-    flipper_post_install(installer)
-+    installer.pods_project.targets.each do |target|
-+      target.build_configurations.each do |config|
-+        config.build_settings['APPLICATION_EXTENSION_API_ONLY'] = 'NO'
-+      end
-+    end
-  end
-end
-
-target '<SHARE_EXTENSION_NAME>' do
-  config = use_native_modules!
-
-  use_react_native!(:path => config["reactNativePath"])
-end
-```
-
-Run `pod install` in your `ios/` directory.
-
 Right click on your Share Extension folder, and choose `Add Files to "ProjectName"`
 
 On the pop-up, select `node_modules/react-native-share-menu/ios/ReactShareViewController.swift`. Make sure `Copy items if needed` is NOT selected and that the selected target is your newly created Share Extension
@@ -274,7 +286,10 @@ Feel free to change the values in ReactShareViewBackgroundColor to whatever you 
 Finally, in your `index.js` file, register the component you want to render in your Share Extension view:
 
 ```javascript
-AppRegistry.registerComponent('ShareMenuModuleComponent', () => MyShareComponent);
+AppRegistry.registerComponent(
+  "ShareMenuModuleComponent",
+  () => MyShareComponent
+);
 ```
 
 If you're rendering an empty component, you should be seeing something similar to this when you share to your app:
@@ -353,11 +368,11 @@ const Test = () => {
 };
 
 const Share = () => {
-  const [sharedData, setSharedData] = useState('');
-  const [sharedMimeType, setSharedMimeType] = useState('');
+  const [sharedData, setSharedData] = useState("");
+  const [sharedMimeType, setSharedMimeType] = useState("");
 
   useEffect(() => {
-    ShareMenuReactView.data().then(({mimeType, data}) => {
+    ShareMenuReactView.data().then(({ mimeType, data }) => {
       setSharedData(data);
       setSharedMimeType(mimeType);
     });
@@ -393,11 +408,13 @@ const Share = () => {
       <Button
         title="Continue In App With Extra Data"
         onPress={() => {
-          ShareMenuReactView.continueInApp({hello: "from the other side"});
+          ShareMenuReactView.continueInApp({ hello: "from the other side" });
         }}
       />
       {sharedMimeType === "text/plain" && <Text>{sharedData}</Text>}
-      {sharedMimeType.startsWith("image/") && <Image source={{uri: sharedData}} />}
+      {sharedMimeType.startsWith("image/") && (
+        <Image source={{ uri: sharedData }} />
+      )}
     </View>
   );
 };

